frontend/src/components/single-contract/CloseJobFormButton.tsx

```diff
@@ -0,0 +1,57 @@
+import { CloseJobModal } from 'components/smart-contracts/modals/CloseJobModal';
+import { Form, Formik } from 'formik';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useSingleContract } from './context/useSingleContract';
+
+export const CloseJobFormButton = () => {
+  const { data: job } = useSingleContract();
+  const [showCloseJobModal, setShowCloseJobModal] = useState(false);
+  const history = useHistory();
+
+  return job ? (
+    <Formik
```

Comment:
It's fine to use Formik here if you want, but I'd probably just keep it simple with an `onClick` and `useState`, especially since we're not dealing with any actual values.

```
const [showModal, setShowModal] = useState(false);

<button onClick={() => setShowModal(true)}>Request...</button>
```

Reply:
Agreed.  There are a few other forms like this one, so we can clean that up in a later PR.

Reply:
Added #109